runtime/src/accounts.rs

```diff
@@ -155,7 +155,8 @@ impl Accounts {
                             AccountsDB::load(storage, ancestors, accounts_index, key)
                                 .map(|(mut account, _)| {
                                     if message.is_writable(i) && !account.executable {
```

Comment:
Well, these inconsistent predicates are bothersome. Ideally, I want to put everything into `RentCollector::update`/`RentCollector::from_existing_account()`. But, to avoid any additional risks, I've punted it. ;)